examples/java/src/main/resources/META-INF/services/org.apache.beam.sdk.harness.JvmInitializer

```diff
@@ -0,0 +1 @@
+org.apache.beam.examples.common.ExampleJVMInitializer
```

Comment:
You don't need to do this manually, I think.

Reply:
Yes, if we use @AutoService this file will be generated automatically.